packages/client/src/FC/Base/FC__Button.re

```diff
@@ -77,39 +89,38 @@ let styles =
       ])
     );
 
-  switch (isDisabled, fullWidth) {
-  | (false, false) => main
-  | (true, false) => Css.merge([main, disabledStyles])
-  | (false, true) => Css.merge([main, fullWidthStyle])
-  | (true, true) => Css.merge([main, disabledStyles, fullWidthStyle])
-  };
+  let total =
+    switch (isDisabled, fullWidth) {
+    | (false, false) => main
+    | (true, false) => Css.merge([main, disabledStyles])
+    | (false, true) => Css.merge([main, fullWidthStyle])
+    | (true, true) => Css.merge([main, disabledStyles, fullWidthStyle])
+    };
+
+  Css.merge([sizeStyles(size), total]);
 };
 
+// Button must be button
 let component = ReasonReact.statelessComponent(__MODULE__);
-
 let make =
     (
-      ~href=?,
       ~onClick=?,
       ~variant=Secondary,
+      ~size=Large,
       ~isDisabled=false,
       ~fullWidth=false,
-      ~verticalPadding=`px(4),
       ~className="",
       children,
     ) => {
   ...component,
   render: _self =>
-    <FC__Link
```

Comment:
I thought the FC__Link had some fancy-ish href properties. Are any of those needed?

Reply:
Sorry, did not get properly.

Reply:
Never mind; I thought it did some things it doesn't actually do.